File: src/main/java/mvp/UserFragmentMain.java
package mvp;

import mvp.user.UserFragment;

public class UserFragmentMain {

    public static void main(String[] args) throws InterruptedException {
        UserFragment userFragment = new UserFragment();

        userFragment.onCreate();
        Thread.sleep(400);
        userFragment.onCreateView();
        Thread.sleep(400);
        userFragment.onDestroyView();
        Thread.sleep(400);
        userFragment.onDestroy();

        System.out.println("PROGRAM FINISHED");
    }
}

File: src/main/kotlin/chapter2OOP/mvp/UserFragmentMain.kt
package chapter2OOP.mvp

import chapter2OOP.mvp.user.UserFragment


fun main() {
    val userFragment = UserFragment()
    userFragment.onCreate()
    Thread.sleep(400)

    userFragment.onCreateView()
    Thread.sleep(400)

    userFragment.onDestroyView()
    Thread.sleep(400)
    userFragment.onDestroy()

    println("PROGRAM FINISHED")
}
